load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -371,16 +372,16 @@ func (plans *actorLoadTestIncrementPlans) validateMaxAmounts(t gocuke.TestingT)
 // totalDurationBlocks returns the number of blocks which will have elapsed when the
 // proof corresponding to the session in which the maxActorCount for the given actor
 // has been committed.
-func (plans *actorLoadTestIncrementPlans) totalDurationBlocks(sharedParams *sharedtypes.Params) int64 {
+func (plans *actorLoadTestIncrementPlans) totalDurationBlocks(
+	sharedParams *sharedtypes.Params,
+	currentHeight int64,
+) int64 {
 	// The last block of the last session SHOULD align with the last block of the
 	// last increment duration (i.e. **after** maxActorCount actors are activated).
 	blocksToLastSessionEnd := plans.maxActorBlocksToFinalIncrementEnd()
```

Comment:
Optional consideration: s/last/prev

Ditto below

Reply:
Thanks for calling out the ambiguity around "last" here. I think in this case it should be "final".